src/Angor/Shared/Services/RelaySubscriptionsHandling.cs

```diff
@@ -102,6 +102,14 @@ public void HandleEoseMessages(NostrEoseResponse _)
         if (!relaySubscriptions.ContainsKey(_.Subscription)) 
             return;
         
+        if (_.Subscription.EndsWith("DM") || 
```

Comment:
ahh this is a very very ugly hack lol

Instead we should allow the component to create a subscription that will not close until it is disposed.